tests/source_type_test.py

```diff
@@ -42,7 +42,7 @@ class WindowsRegistryKeySourceTypeTest(unittest.TestCase):
 
   def testInitialize(self):
     """Tests the __init__ function."""
-    source_type.WindowsRegistryKeySourceType(keys=[u'test'])
+    source_type.WindowsRegistryKeySourceType(keys=[ur'HKEY_LOCAL_MACHINE\test'])
```

Comment:
python3 doesn't like ur (see travis failure). Probably because all strings are unicode in 3. Use \\ here and below instead. Use tox to test it locally.

Lets also add a failure test where the keys is a string.